pennylane/devices/legacy_facade.py

```diff
@@ -98,6 +103,15 @@ def legacy_device_batch_transform(tape, device):
     return _set_shots(device, tape.shots)(device.batch_transform)(tape)
 
 
+def adjoint_ops(op: qml.operation.Operator) -> bool:
+    """Specify whether or not an Operator is supported by adjoint differentiation."""
+    if isinstance(op, qml.QubitUnitary) and not qml.operation.is_trainable(op):
+        return True
+    return not isinstance(op, MidMeasureMP) and (
```

Comment:
In anticipation of #5868 

```suggestion
    return not isinstance(op, (MidMeasureMP, qml.ops.Conditional)) and (
```

Reply:
Wait ignore this. This should be resolved in the same PR, not here.